packages/api/src/api/helix/moderation/HelixModerationApi.ts

```diff
@@ -286,4 +305,65 @@ export class HelixModerationApi extends BaseApi {
 
 		return result.data.map(settingsData => new HelixAutoModSettings(settingsData));
 	}
+
+	/**
+	 * Bans or times out a user in a channel
+	 *
+	 * @param broadcasterId The ID of the broadcaster in whose channel the user will be banned/timed out.
+	 * @param moderatorId The ID of a user that has permission to ban/timeout users in the broadcaster's chat room.
+	 * This must match the user ID associated with the user OAuth token.
+	 * @param data The data about the ban/timeout, including the user's ID, the reason, and the duration (for timeouts only).
+	 * Don't use the "data" wrapper.
```

Comment:
The documentation already shows the exact format, so this is unnecessary.

Reply:
Fixed.

Reply:
Just saw the second comment. Will update.

Reply:
Fixed.